dbt/include/bigquery/macros/materializations/seed.sql

```diff
@@ -10,8 +10,8 @@
 {% macro bigquery__load_csv_rows(model, agate_table) %}
 
   {%- set column_override = model['config'].get('column_types', {}) -%}
-  {{ adapter.load_dataframe(model['database'], model['schema'], model['alias'],
-  							agate_table, column_override) }}
+    {{ adapter.load_dataframe(model['database'], model['schema'], model['alias'],
```

Comment:
I'd recommend keeping the same indentation as before unless there is a reason or need to change it:
```suggestion
  {{ adapter.load_dataframe(model['database'], model['schema'], model['alias'],
```